ui/views.py

```diff
@@ -57,7 +57,8 @@ def get(self, request, *args, **kwargs):
             "name": name,
             "username": username,
             "host": webpack_dev_server_host(request),
-            "edx_base_url": settings.EDXORG_BASE_URL
+            "edx_base_url": settings.EDXORG_BASE_URL,
+            "search_url": "http://localhost:9200",
```

Comment:
I changed `docker-compose.yml` to forward `localhost:9100` to the ES docker container and changed the search url accordingly (so the browser knows what to hit). Could you pull the latest and check to see if it works ok with devstack / on mac os?